pkg/util/validation/limits.go

```diff
@@ -129,6 +130,7 @@ type Limits struct {
 	MaxTotalQueryLength                    model.Duration `yaml:"max_total_query_length" json:"max_total_query_length"`
 	ResultsCacheTTL                        model.Duration `yaml:"results_cache_ttl" json:"results_cache_ttl" category:"experimental"`
 	ResultsCacheTTLForOutOfOrderTimeWindow model.Duration `yaml:"results_cache_ttl_for_out_of_order_time_window" json:"results_cache_ttl_for_out_of_order_time_window" category:"experimental"`
+	MaxQuerySizeBytes                      int            `yaml:"max_query_size_bytes" json:"max_query_size_bytes" category:"experimental"`
```

Comment:
What do you think about "query expression size bytes" or "query expression length bytes" to clarify it's not the query execution but the actual query expression?

Reply:
I went with "query expression size bytes" since we already use "length" to refer to the time range of a query.